src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/PrisonApi.java

```diff
@@ -313,21 +311,6 @@ public void deleteAppointment(final Long appointmentId, final EventPropagation p
             .block();
     }
 
-    public void updateAppointmentComment(long appointmentId, UpdateComment comment, final EventPropagation propagation) {
-        webClient.put()
-            .uri("/appointments/{appointmentId}/comment", appointmentId)
-            .header("no-event-propagation", propagation.doNotPropagate())
-            .contentType(MediaType.APPLICATION_JSON)
-            .bodyValue(comment)
-            .retrieve()
-            .toBodilessEntity()
-            .onErrorResume(WebClientResponseException.NotFound.class, notFound -> {
-                logger.info("Ignoring appointment with id: '{}' that does not exist in nomis", appointmentId);
-                return Mono.empty();
-            })
-            .block();
-    }
-
```

Comment:
Was this ONLY used for VLB comment updates? Don't suppose any other appointment had an edit..

Reply:
Only used by VLB related code.

In fact if look at the signature of the removed function, the `UpdateComment` type belongs to the VLB court code base.